src/lib/ThumbnailsSidebar.js

```diff
@@ -81,15 +84,38 @@ class ThumbnailsSidebar {
             const { bpPageNum: pageNumStr } = thumbnailEl.dataset;
             const pageNum = parseInt(pageNumStr, 10);
 
-            if (this.onClickHandler) {
-                this.onClickHandler(pageNum);
+            if (this.onThumbnailSelect) {
+                this.onThumbnailSelect(pageNum);
             }
         }
 
+        this.anchorEl.focus();
         evt.preventDefault();
         evt.stopImmediatePropagation();
     }
 
+    /**
+     * Method to handle keyboard events in the Thumbnails Sidebar
+     *
+     * @param {Object} evt - Key event
+     * @return {void}
+     */
+    onKeydown(evt) {
```

Comment:
Can we use `event` here? We've been trying to avoid abbreviations where possible.